components/suggestion/at_mention_provider/at_mention_provider.jsx

```diff
@@ -99,6 +99,11 @@ export default class AtMentionProvider extends Provider {
         return groupSuggestions;
     }
 
+    // normalizeSuggestions removes accents from strings to match ascii encoding
+    normalizeSuggestions(name) {
```

Comment:
I changed the name but I kept the comment simple though. Because normalize is a vanilla js function I think it should be clear and easy for someone to figure out the purpose of the method.

Reply:
Makes sense to me. It's definitely good to explain stuff like that which is a bit different from what someone else reading the code might expect